src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products/products.yml

```diff
@@ -31,20 +31,11 @@ _product_warehouses:
 
 # Product List
 admin_product_catalog:
-    path: /{offset}/{limit}/{orderBy}/{sortOrder}
+    path: /
```

Comment:
If we validate this change, there is a BC break.

Reply:
Yes indeed, however if someone has a link toward this page:
- if the link has been generated by Symfony (like in twig `path('admin_product_catalog', {'offset': my_offset, ...)` it works as the parameters are added as GET parameters instead of in the path
- if the link is hardcoded it is wrong indeed 😢 

This BC break has been discussed with @PierreRambaud and approved (for the better we hope)
@PierreRambaud can you confirm as my memory is a bit blurry about this ? 😅

Reply:
It was approved with @eternoendless in a previous discussion. It's a BC only if the link is hardcoded and this url was added recently: #9137

Reply:
I agree with you, but in the previous PR we kept the old URL as well.

Is there an issue by making both URLs working?